components/dashboard/src/settings/Billing.tsx

```diff
@@ -13,15 +13,16 @@ import { AttributionId } from "@gitpod/gitpod-protocol/lib/attribution";
 
 export default function Billing() {
     const { user } = useContext(UserContext);
-    const attributionId: AttributionId = { kind: "user", userId: user?.id || "" };
 
     return (
         <PageWithSettingsSubMenu title="Billing" subtitle="Usage-Based Billing.">
```

Comment:
Note: This is the title shown at the top of the page. Also, I left the section headings in the body of the page for now, but I'm happy to remove them (I wasn't quite sure if they were still needed, because the specs only focus on the contents, and not the general context around them -- so I didn't see if there was still a title above it or not)

Reply:
> I wasn't quite sure if they were still needed, because the specs only focus on the contents, and not the general context around them

Ah, ok! I thought it was clear that these specs included only the parts we're changing (see [relevant discussion](https://gitpod.slack.com/archives/C01KPEPQYE6/p1664795185245119?thread_ts=1664363685.869519&cid=C01KPEPQYE6) (internal)), but you've got a valid point, there was no decision documented around dropping the section headers.

Let's keep them for now and see if someone complains about this change, we can always drop them later, which would take less than 5 minutes to make, see [relevant discussion](https://gitpod.slack.com/archives/C01KPEPQYE6/p1664538609541969?thread_ts=1664363685.869519&cid=C01KPEPQYE6) (internal). 🤝

Reply:
Thanks! I'm more than happy to delete the titles. However, I get lost in all our discussions, meeting notes, recordings, Notion pages, and PR comments 😅 

Could you please take a few quick screenshots, and obviously highlight what needs to be removed? Then it will be really easy for me to do so.

Reply:
No need to change anything for now. Let's rather keep the section headers as they are, as removing the section headers requires us to introduce a better dropdown element for the billing account. Dropping the headers now would be quite confusing. 🤝